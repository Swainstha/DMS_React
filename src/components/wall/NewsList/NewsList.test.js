import React from 'react';
import NewsList from './index';

const dummy_data={
    newsList: [{id:3},{id:1},{id:2}]
}
import testSnapFunction from '../../../util/testSnapFunction';
testSnapFunction("<NewsList/>", "Snapshot test for NewsList",<NewsList {...dummy_data}/> )
