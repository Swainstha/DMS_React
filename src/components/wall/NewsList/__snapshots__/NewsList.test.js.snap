// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewsList/> Snapshot test for NewsList 1`] = `ShallowWrapper {}`;
